setup.py

```diff
@@ -22,7 +22,7 @@
 extras_requirements = {"dask": ["dask", "distributed"]}
 
 requirements = [
-    "qiskit>=0.45.2",
+    "qiskit>=1.1.0",
```

Comment:
Is this needed for documentation?
If so, we have to discuss we can upgrade qiskit or not before merging. 
(We are planning to upgrade in Aer 0.15 release but I think it is too early to upgrade to qiskit 1.1)

Reply:
This is Suggested by @kevinsung that's why I just updated. 
Anyway I don't think there should be any problem, I have made all the tutorials compatible to `qiskit 1.1`

Reply:
Perhaps I misunderstood the issue. I thought that the code updates in the tutorials would require Qiskit 1.0.

Reply:
I confirm that we can merge setup.py with Qiskit 1.1 requirement